### DISCUSSION TITLE
ASTAP Settings

### DISCUSSION THREAD

**Discussion by JDAstroPhoto** [mworion/MountWizzard4#270] [Q&A] [answered]
I consistently get about 4 to 8 points out of 65 - 85 that are way, way off on the plate solve.  1000's of seconds off.  Each point will take the residual from 3.4 to 1500 or more.  MW4 runs great, and the points turn green.  
It's not the sky, since I ran it Mon, Wed and Fri this week at the SDAA Dark Sky facility and the sky was perfectly clear.  It is also not the radius search area, since I played with that already.  I pumped it up to 20 degrees.  
I wonder if I have some corrupt Plate Solve files?  I think I will reload those files.  Does anyone else seen this?  Are there any other settings for ASTAP?
Thanks

**Comment by mworion** [comment]
Hi,
sound like a strange behavior. I hardly believe that it might be a corrupt file. You're telling it's consistent and reproducible(?). Could you post the model file or a picture of the model. If you set the loglevel within MW4 to debug (under settings/log) and post such a model run, I might be able to look on the solving results and we could find out what happens there. Are the points close to zenith or other "special" places ?
Michel

**Comment by JDAstroPhoto** [comment]
[mw4-2022-08-27.log](https://github.com/mworion/MountWizzard4/files/9440095/mw4-2022-08-27.log)
Thanks Michel.
This was done at SDAA Tierra Del Sol, Dark Sky spot.  Not a cloud in the sky, perfect sky.   I limited MW4 to 25 degrees on the horizon.  So no points lower than 25 degrees.
Attached is the log, it is very long.  
1. The first model is a 35 point model I used to get my polar alignment even closer.  You can see there were 8 points that had 52,000 RMS errors.  Once I deleted those points, my 27 point model was around 4 RMS.
2. Then you will see pages and pages of imaging.  I use your excellent Polar alignment "image" tab to adjust polar alignment.  I do have problems with "Expose N".  It takes a very long time between display of exposures.  I think the exposures are being made, however, they don't display right away on the "image" tab of your excellent MW4.  Maybe you can help me with that as well.  It takes over 30 seconds and sometimes doesn't display at all.  I then usually hit "Expose N' again.  Or I "Abort" and hit it again.  Either way, it takes a very long time to display the exposed images.  I believe the images are being taken as fast as possible, however, MW4 is not displaying them for some reason.  Maybe it exposes 1 out of 10 or 20.  I've used both 1 second exposures and 2 second exposures, it makes no difference.
3. Then it was time to run the 83 point model.  There are some false starts, where the camera disconnected, I did some restarts until NINA and MW4 were in sync.
4. The 83 point model completes.  
5. There were 5 points with RMS of 52,000
6. I delete those points for a 77 point model.  The RMS is around 3.7
7. I used that model for the rest of the night.  Eccentricities of stars were great, 0.4 - 0.7 or so.  No complaints.  Excellent model.

I figured this behavior was normal, just bad plate solves that happens with ASTAP.  But when I mentioned this on the 10micron forum, members thought I was chasing RMS.   It doesn't seem like anyone else on the 10micron forum has this handful of corrupt plate solves.
Thanks, I really appreciate you help

**Comment by JDAstroPhoto** [comment, marked as answer]
Attached is the Analysis of the 83 pt model.
![AltAZ_83pt](https://user-images.githubusercontent.com/111151828/187086614-3ac84730-edeb-48c4-bab9-4c83f21fb0d6.JPG)
![General_83pts](https://user-images.githubusercontent.com/111151828/187086615-a35f4ea0-765e-4c25-af9c-3168bd4ab6ea.JPG)
![RADEC_83pt](https://user-images.githubusercontent.com/111151828/187086616-af06713b-bb88-4017-ac09-e462815b5566.JPG)

**Reply by mworion** [reply]
Hi,
you were right to delete the five points in your final model. As you could see in the analyse window, first tab, upper right chart, these were really out of nowhere. So from my point of view not chasing RMS -> If you port this charts on 10micron forum, the others will agree.

To you setup (as I image also with an QHY600) so large image sensors. I would recommend at least bin2 (you might go for bin4 as well) and or limit to frame to 70-80%. ASTAP needs in average 1500 Pix to solve good, too much more is bad, less as well.
I go for 3-5 s exposures, which is normally good. With these settings download times improve much. You don't need full resolution in solving anyway.

Normally you should be not able to start exposeN once you started it. Also for polar alignment I would go for the same settings. I addressed this for the next release.

Still one task open, why extreme points happen. If you keep all images of the model or polar model (checkBox in imaging tab), please look on the images if there is something to discover (star trails, hardly any stars, ....) and to find out.

Michel

**Reply by JDAstroPhoto** [reply]
Thanks so much, that's a very logical and great response.  Yes, 1x1 binning is 120MB.  I am heading out tonight again for SDAA (TDS) so I will give the binning a try.  I have a feeling that might be it, just too much data.
I have 80 GIG of images, but for some reason I don't have the 85 plate solves for the model, maybe MW4 doesn't save those automatically?  Is there a parameter I need to set to keep the 85 pt model plate solve images?  I have all the Polar Alignment images.  So I was correct in my presumption.  The camera was taking pictures like crazy, they were just not getting displayed on "Image".  Since this happens to me every time, I like your idea, way too much data.

**Reply by Ruediger3** [reply]
I use the ASI6200 - same sensor:
Binning 2, 4s exposure at gain 100 (not directly comparable to QHY), 100% crop., settle time 3s

I have constantly 100% solve rate. Model has 90 points. 

Cheers
Rüdiger

**Reply by mworion** [reply]
Unfortunately all images on „ExposeN“ are saved. So you could delete them all (will be changed in next version to decide wether to keep). I do bin2 and 50% subframe.
Yes, you need to check the checkbox „keep all images from modeling“ in imaging tab (where you set the exposure times as well). The images will be save in a subfolder named like you model and numbered in there like the order of your model points.
Michel

**Reply by JDAstroPhoto** [reply]
Thanks, perfect, I will try it tonight, this is great news, since it will save me setup time.  Particularly for the Polar alignment routine.

**Reply by JDAstroPhoto** [reply]
Thanks Ruediger3, Binning 2, 4s.
Do you use the low gain or high gain for the sensor?  My guess would be high gain, so for QHY it is 56 gain.

**Reply by Ruediger3** [reply]
Hi JD,
the ASI does not know the both gain modes. It switches automatically at gain = 100.

Happy to hear you had success ;)

Cheers
Rüdiger

**Comment by JDAstroPhoto** [comment]
Success,  I went with 2x2 Bin and 50% subframe.  Worked like a charm, and Polar alignment "Image" was 3 or 4 times more responsive.  I am very happy with the results.  Below is the 76 pt model.  There was one point in the EAST that had a higher RMS, as you can see from the graph.  There are monsoons in Arizona, and we have thunderheads in the East.  So it didn't surprise me.  Since I wasn't going to image there anyway last night, I deleted one point and went with the 75 pt model.  
Thanks Michel and Ruediger3 for your help.  I appreciate it.

![MW4_76pts](https://user-images.githubusercontent.com/111151828/187284938-037c761e-08aa-4763-ae01-15b48282a2a9.JPG)

**Comment by mworion** [comment]
You‘re welcome. Great that it worked.
The bullet proof comes with the first images taken with that model…
Michel
